plugins/webp-uploads/picture-element.php

```diff
@@ -168,3 +168,20 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		$image
 	);
 }
+
+/**
+ * Adds CSS to make Lightbox trigger BUTTON visible on an Image lightbox when Picture Element is enabled.
+ *
+ * @since 2.4.0
+ * 
+ */
+function webp_make_lightbox_trigger_button_visible() {
+    if ( has_filter( 'render_block_core/image', 'block_core_image_render_lightbox' ) ) {
+		echo '<style>
+			.wp-lightbox-container > :hover {
```

Comment:
Since we're specifically targeting the Picture element here, might as well make this specific to `PICTURE`. More importantly, this is missing the next-child `+ button` selector:

```suggestion
			.wp-lightbox-container picture:hover + button {
```

Reply:
Used `.wp-lightbox-container > picture + button` as adding `:hover` to picture was not applying the CSS when tested. Let me know it the updated one looks good.